php-80/Dockerfile

```diff
@@ -21,15 +21,6 @@ RUN set -xe \
  && yum groupinstall -y "Development Tools" --setopt=group_package_types=mandatory,default
 
 
-# The default version of cmake is 2.8.12. We need cmake to build a few of
-# our libraries, and at least one library requires a version of cmake greater than that.
-# Needed to build:
-# - libzip: minimum required CMAKE version 3.0.
-RUN LD_LIBRARY_PATH= yum install -y cmake3
-# Override the default `cmake`
-RUN ln -s /usr/bin/cmake3 /usr/bin/cmake
```

Comment:
cmake3 is installed by default

Reply:
This is due to AL2 having both `cmake` and `cmake3` packages to ensure that software needing `cmake` that didn't yet work with `cmake3` could continue to be built.

For AL2023, it's all `cmake` version 3.